hugegraph-llm/src/tests/operators/llm_op/test_keyword_extract.py

```
@@ -0,0 +1,284 @@
+        self.assertEqual(result["query"], self.query)
+
+    def test_run_with_no_query_raises_assertion_error(self):
+        """Test run method with no query raises assertion error."""
+        # Create extractor with no query
+        extractor = KeywordExtract(llm=self.mock_llm)
+
+        # Create context with no query
+        context = {}
+
+        # Call the method and expect an assertion error
+        with self.assertRaises(AssertionError) as context:
```

Ruff (0.11.9)

Redefinition of unused `context` from line 146

(F811)